lib/wallet/src/Cardano/Wallet/Primitive/Types/UTxOStatistics.hs

```diff
@@ -119,9 +119,9 @@ log10 = Log10
 {-# INLINE log10 #-}
 
 -- | Computes a 'UTxOStatistics' object from a 'UTxO' set.
```

Comment:
Mention that `log10` is used in the doc comment?

Reply:
> Mention that `log10` is used in the doc comment?

Thanks for the suggestion: I'll make a follow up PR with this!